### PR TITLE
Update mattermost-templated-agreements.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Updated the process to have internal signatures route through Procurement Team vs. HR.

### CODE REVIEW

File: operations/finance/risk-management/mattermost-templated-agreements.md
Comment (justinegeffen) [inline operations/finance/risk-management/mattermost-templated-agreements.md]:
```suggestion
4. HR will review the document to ensure the [criteria](https://handbook.mattermost.com/operations/finance/risk-management/mattermost-templated-agreements#criteria) are met and will forward to the [Procurement Team](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-4-send-for-internal-signature) for internal [signatures](https://handbook.mattermost.com/operations/operations/company-agreements#who-can-sign-on-behalf-of-the-company) and CC the [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations).  
```

File: operations/finance/risk-management/mattermost-templated-agreements.md
Comment (justinegeffen) [inline operations/finance/risk-management/mattermost-templated-agreements.md]:
```suggestion
5. Upon receipt of the fully executed agreement please follow [Procurement Step 7](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-7-vendor-on-boarding-and-invoice-processing) to ensure your vendor is set up for payment.  
```

File: operations/finance/risk-management/mattermost-templated-agreements.md
Comment (justinegeffen) [inline operations/finance/risk-management/mattermost-templated-agreements.md]:
```suggestion
2. The appropriate [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations) should complete the [Mattermost U.S. Consulting Agreement Template](https://app.hellosign.com/s/GnvOJVYW). Reference the [field description guide](https://docs.google.com/document/d/1FKef8xf9N_NTEc0owbAGH4mb0UzU8CJQ5qckGvIl1UM/edit) for help. 
```

File: operations/finance/risk-management/mattermost-templated-agreements.md
Comment (justinegeffen) [inline operations/finance/risk-management/mattermost-templated-agreements.md]:
```suggestion
3. Upon completion, the CAO will receive the document via HelloSign. Forward that email to [HR](mailto:%20hr@mattermost.com) and include your vendor's signatory contact information \(full name and signature\). 
```

File: operations/finance/risk-management/mattermost-templated-agreements.md
Comment (justinegeffen) [inline operations/finance/risk-management/mattermost-templated-agreements.md]:
```suggestion
1. Confirm you have [budget approval](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval) to engage with a vendor.
```

File: operations/finance/risk-management/mattermost-templated-agreements.md
Comment (justinegeffen) [inline operations/finance/risk-management/mattermost-templated-agreements.md]:
```suggestion
2. The appropriate [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations) should complete the [Mattermost U.S. Consulting Agreement Template](https://app.hellosign.com/s/GnvOJVYW). Reference the [field description guide](https://docs.google.com/document/d/1FKef8xf9N_NTEc0owbAGH4mb0UzU8CJQ5qckGvIl1UM/edit) for help. 
```

**Review by justinegeffen — APPROVED** [review]
Looks good to me! Thank you!

For the record, I made some editorial changes and committed them: Removed extraneous spaces and inserted missing spaces in the URL formatting, removed a hyphen from "set up", and edited "Vendor" to "vendor".

**Comment by jasonblais** [comment]
@natalie-hub Thank you! Let us know if this should get reviewed by anyone before merging? (e.g. Tim or Aneal)

**Comment by justinegeffen** [comment]
Hi @natalie-hub. Following up on this to find out whether it should have an additional review by @TQuock before we merge it?

**Comment by natalie-hub** [comment]
@justinegeffen - I'm so sorry I missed this note. I actually updated this while I was on a call with @TQuock and he and I drafted this on the call together. 

Thank you for pointing out, though. In the future, I will add @TQuock as a PR.

**Comment by justinegeffen** [comment]
Thanks, @natalie-hub! Then I think this is good to merge?

**Comment by natalie-hub** [comment]
@justinegeffen - yes, this is good to merge. Thank you!

**Pull request merged**
